docker/build.sh

```diff
@@ -93,7 +93,7 @@ CMD ["sh", "-c", "docker/scripts/build.sh ${ARGS}"]
 EOF
 
 # GREMLIN_SERVER is the project version e.g. 3.5.5-SNAPSHOT
-export GREMLIN_SERVER=$(grep tinkerpop -A2 pom.xml | grep -Po '(?<=<version>)([0-9]+\.?){3}(-SNAPSHOT)?(?=<)')
+export GREMLIN_SERVER=$(grep tinkerpop -A2 pom.xml | sed -r -n 's/.*<version>(([0-9]+\.?){3})(-SNAPSHOT)?<\/version>/\1\3/p')
```

Comment:
Nice! I'm a happy macOS-default-grep user now. Could you also make this change to the gremlin-go `run.sh` file? Thanks!

Reply:
Sure, I'll make a separate PR for this.